sys/vm/vm_mmap.c

```diff
@@ -152,23 +152,30 @@ cap_covers_pages(const void * __capability cap, size_t size)
 			    CHERI_PERM_STORE_LOCAL_CAP)
 #define	PERM_EXEC	CHERI_PERM_EXECUTE
 #define	PERM_RWX	(PERM_READ | PERM_WRITE | PERM_EXEC)
+
 /*
- * Given a starting set of CHERI permissions (operms), set (not AND) the load,
- * store, and execute permissions based on the mmap permissions (prot).
+ * mmap_prot2perms - convert an mmap protection value to capability
+ * permissions.
  *
- * This function is intended to be used when creating a capability to a
- * new region or rederiving a capability when upgrading a sub-region.
+ * This function implies from PROT_CAP_* from PROT_READ and PROT_WRITE
```

Comment:
```suggestion
 * This function implies PROT_CAP_* from PROT_READ and PROT_WRITE
```

Possibly, s/implies/derives/?